### ISSUE TITLE
Add simple tool to extract and cluster output of fully interacting replica

### ISSUE THREAD

**Issue by jchodera** [choderalab/yank#154]

**Comment by jchodera** [comment]
I think some basic tool here would be important for a first release.  The simplest approach is to write a tool that just collects all data from the fully interacting thermodynamic state and aligns the receptor atoms to the first frame, dumping this to a trajectory or multimodel PDB file.  

Hopefully, this is easy to do with `mdtraj`.

Is it something we could get, for example, Andrea to put together?

**Comment by kyleabeauchamp** [comment]
Yes.  We should also examine https://github.com/choderalab/yank/issues/73, which may cuase problems here

**Comment by luirink** [comment]
I have a script I wrote (which I can rewrite such that it is more readable/useable for everyone) which collects the data from the fully interacting state, aligns the alpha-carbon of the receptor to first frame. It then continues with a clustering of the ligand trajectory only, by performing first a PCA (sklearn PCA code) and then kmeans clustering (sklearn code). It writes out the explained variance for the PCA and kmeans. Furthermore, it prints out the medoid frames in xtc format, xtc trajectories of the clusters, and some kmeans  and pca images. Would that be useful? On code). It writes out the explained variance for the PCA and kmeans. Furthermore, it prints out the medoid frames in xtc format, xtc trajectories of the clusters, and some kmeans  and pca images. Would that be useful?  
Dumping the fully interacting trajectory as total in addition to the clusters trajectories (which include all interacting frames) could be easily implemented as well...

**Comment by kyleabeauchamp** [comment]
It might make sense to use msmbuilder for some of this.  http://msmbuilder.org/latest/

**Comment by jchodera** [comment]
We can likely use msmbuilder clustering schemes:
http://msmbuilder.org/latest/cluster.html#cluster

**Comment by jchodera** [comment]
Would also be useful to allow the user to specify clusters in a file.

**Comment by hlfsimko** [comment]
Would it be possible to get a hand on the (unfinished) script? Could you maybe add it to the repo?

**Comment by jchodera** [comment]
@luirink has generously given us a copy of her clustering code, so we can work on incorporating this early next week.

**Comment by luirink** [comment]
that would be great!

**Comment by Lnaden** [comment]
Tool would need to go in post 1.0 release
